### ISSUE TITLE
hour-view number of columns

### ISSUE THREAD

**Issue by makor-capital** [indrimuska/angular-moment-picker#207]
hi,
it is possible to change the number of column in the hour-view
instead of a table of 4 columns, to have a table of 6 columns (so then have less rows)
thanks
![untitled-3](https://user-images.githubusercontent.com/22317564/29114135-f8e3ef54-7cfb-11e7-9f30-cd3ba059cdc6.jpg)

**Comment by andreashouben** [comment]
Yes, that'd be great. It's perfect in the selection used for seconds.

**Comment by andreashouben** [comment]
#247 and #181 offer solutions for this problem. Please merge one of them :-)
